### ISSUE TITLE
Display index.html

### ISSUE THREAD

**Issue by rAlexandre00** [playfultechnology/propcontrol#8]
Could you provide the index.html file (and other needed files) that are used for the in-game display?

**Comment by AshbyGeek** [comment]
I second this request. The wiki page articles are useful, and the contents of  the various json files in the NODE-Red folder can be viewed, but its a little hard to follow the flow of logic without the accompanying web page files.

**Comment by gabrielcor** [comment]
index.html - https://pastebin.com/WwgkGpqx
index.js -  https://pastebin.com/bP1Vqx7T
index.css - https://pastebin.com/j6qfTy9h

**Comment by alastaira** [comment]
Yeah, I accept that it's a little annoying that the uibuilder node relies on static HTML/JS/CSS files which are served from the filesystem and therefore their contents aren't contained within the flow itself. I'd considered adding their contents as comments within the flow, but the risk is that they are not maintained and become out-of-sync.
I'm currently in the process of rewriting a lot of the dashboard code to use the latest version of the uibuilder node, so will see if there now exists a better way of storing that information within the flow.
